widgets/CMakeLists.txt

```diff
@@ -116,6 +116,12 @@ if (SYNCTHING_WIDGETS_LOG_JAVASCRIPT_CONSOLE)
     message(WARNING "JavaScript console of web view will be logged to stderr")
 endif ()
 
+# configure autostart .desktop file exec path
+option(AUTOSTART_EXEC_PATH "Set the Exec= path for Linux' autostart/syncthingtray.desktop file - useful for Nix & Guix")
+if (AUTOSTART_EXEC_PATH)
+    add_compile_definitions(AUTOSTART_EXEC_PATH="${AUTOSTART_EXEC_PATH}")
```

Comment:
The definition should only be added for the source file that needs it and not globally for the whole CMake project.